backend/.gitignore

```diff
@@ -158,3 +158,6 @@ dmypy.json
 
```

Comment:
I would suggest moving the `.gitignore` to the root directory of the project and unifying it.